utils/src/poseidon/poseidon_hash.rs

```diff
@@ -26,7 +26,7 @@ impl<F: PrimeField> Poseidon<F> {
     // and skip_matrices is a (temporary) parameter used to generate secure MDS matrices (see comments in the description of find_poseidon_ark_and_mds)
```

Comment:
on line 25 also `lenght` instead of `length` (can't comment that line)

Reply:
Fixed